src/music/extmidi.cpp

```diff
@@ -151,7 +152,7 @@ void MusicDriver_ExtMidi::DoStop()
 			return;
 		}
 		/* Wait 10 milliseconds. */
-		CSleep(10);
+		std::this_thread::sleep_for(std::chrono::milliseconds(10));
```

Comment:
This replacement is so terribly long, couldn't the CSleep() function stay, maybe as an inline function in `thread.h` implemented via std::this_thread::sleep_for?

Reply:
It could. Matter of taste, I don't really care. Not sure what the others think.

Reply:
Done